### ISSUE TITLE
Singularity object pull from hands buggy

### ISSUE THREAD

**Issue by liction15** [UristMcStation/UristMcStation#427]
The singularity has a chance to pull an object from player's hands but it seems to be buggy and doesn't actually pull the object from the hand as the sprite for it still persists. Additionally the "pulled" item is unable to be dropped either through button presses or resting and examining does not show the item in hand. You can also pick up another item despite the sprite for the "pulled" item still being in your hand. I managed to drop the items by click dragging it onto an empty hand.

**Comment by Glloyd** [comment]
yeah, i had this happen to me the other day. I admin deleted it to get rid
of it. gonna look into this.

On Sun, Apr 10, 2016 at 2:24 AM, liction15 <notifications@github.com> wrote:

> The singularity has a chance to pull an object from player's hands but it
> seems to be buggy and doesn't actually pull the object from the hand as the
> sprite for it still persists. Additionally the "pulled" item is unable to
> be dropped either through button presses or resting and examining does not
> show the item in hand. You can also pick up another item despite the sprite
> for the "pulled" item still being in your hand. I managed to drop the items
> by click dragging it onto an empty hand.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/UristMcStation/UristMcStation/issues/427>
>



-- 
Graham Lloyd
